portal/templates/patients_by_org.html

```diff
@@ -156,7 +157,10 @@ <h4 class="tnth-headline">Patient List</h4>
 
 $(document).ready(function() {
 
-  $("#adminTable").on("sort.bs.table reset-view.bs.table", function() {
+  /*****************
```

Comment:
For large block comments - can you make it more clear it is a comment with a prefix on every line - say '#', so a grep or other glances don't mislead us?